app/views/propertyrepresentation/appoint/appointAgentProperties.scala.html

```diff
@@ -120,28 +120,25 @@
     }
 }
 
-@sortOrder(order: ExternalPropertyLinkManagementSortOrder, isSortField: Boolean) = {
-    @{
-        if(isSortField){
-            order match {
-                case ExternalPropertyLinkManagementSortOrder.ASC => "sort_asc"
-                case ExternalPropertyLinkManagementSortOrder.DESC => "sort_desc"
-                case _ => ""
-            }
-        } else ""
-    }
+@sortOrder(order: ExternalPropertyLinkManagementSortOrder, isSortField: Boolean) = @{
+    if(isSortField){
+        order match {
```

Comment:
the Jira ticket says:

>This specifically applies to the 'aria-sort' values. 
>
>Valid values are:
>
>none
>ascending
>descending
>other

can we keep using `sort_asc` and `sort_desc`? I don't really know what are the changes in this PR? It's just different formatting, right?